features/step_definitions/machine_set.rb

```diff
@@ -4,3 +4,46 @@
   cb_name ||= :machinesets
   cb[cb_name] = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api"))
 end
+
+Given(/^I pick a random machineset to scale$/) do
+  ensure_admin_tagged
+  machine_sets = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api"))
+  cache_resources *machine_sets.shuffle
+end
+
+When(/^I scale the machineset to ([\+\-]?)(\d)$/) do | op, num |
```

Comment:
I have suggested to use `NUMBER` because it will also catch `<%= expression %>` format of specifying the number. But I had a mistake in my original suggestion. It should have been:
```
When /^I scale the machineset to ([\+\-]?)#{NUMBER}$/ do | op, num |
```
Sorry if this caused confusion.